### PR TITLE
Update private-cloud.md for request form 22 Feb

### PR DESCRIPTION
**Pull request by Ceballj**
We are making changes to the "Mendix for Private Cloud" request form therefore the documentation needs to be updated to be consistent. See changes in our stage environment: https://newapp868-accp.mendixcloud.com/index.html

@mruiserrmendix  please review as well to ensure I did not miss anything.

We are expecting to deploy the changes to production by Feb 22.

### CODE REVIEW

**Comment by mruiserrmendix** [comment]
It seems ok to me. All information is there.

File: content/developerportal/deploy/private-cloud.md [hunk 1]
```diff
@@ -86,15 +86,18 @@ You can request an Operator license by doing the following:
...
 
-3. Fill in the required information:
-* The name of the namespace(s) you want to request an Operator license for
-* The name of your company
-* The Mendix for Private Cloud architecture type. See [Connected and Standalone Clusters](#connected-standalone), above, for more information.
-Optionally, leave additional information in the _comment_ field.
+4. Fill in the required information:
 
-4. Save the request.
+    * The name of your company
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud.md:95]: Do you have to fill this again, it is mentioned in the basic information above.

File: content/developerportal/deploy/private-cloud.md [hunk 2]
```diff
@@ -80,55 +80,79 @@ The table below shows the differences between the capabilities for apps deployed
...
-3. For **Hosting Type**, select *Mendix for Private Cloud* and click **Next**
+3. For **Hosting Type**, select *Mendix for Private Cloud* and click **Next**.
 
 4. Provide the required information:
-* The number of environments you require a Runtime license for
-* The Mendix for Private Cloud architecture type. See [Connected and Standalone Clusters](#connected-standalone), above, for more information.
-Optionally, leave additional information in the _comment_ field.
+
+    * The name of your company
+    * License Type: Runtime
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud.md:121]: I assume this should have been `Runtime`. I've changed it anyway.

**Review by MarkvanMents — APPROVED** [review]
Hi,
I've proofread this and restructured the licensing section a little bit. I have left a couple of questions, but don't need the answers before approving it.
Please let me know if you want me to review it further, and when it can be published.

**Comment by MarkvanMents** [comment]
Changes deployed. See [TW-1568]

**Pull request merged**